pkg/apis/imageregistry/v1/types.go

```diff
@@ -66,6 +66,10 @@ const (
 	// StorageIncompleteUploadCleanupEnabled denotes whethere or not the registry storage
 	// medium is configured to automatically cleanup incomplete uploads
 	StorageIncompleteUploadCleanupEnabled = "StorageIncompleteUploadCleanupEnabled"
+
+	// VersionAnnotation reflects the version of the registry that this deployment
+	// is running.
+	VersionAnnotation = "imageregistry.operator.openshift.io/version"
```

Comment:
I used “release.openshift.io/version” in the network operator.  Better to have a convention.

Reply:
works for me, i'll update it in the morning.